client/cli/src/params.rs

```diff
@@ -578,74 +594,12 @@ pub struct RunCmd {
 	pub password_filename: Option<PathBuf>
 }
 
-/// Stores all required Cli values for a keyring test account.
-struct KeyringTestAccountCliValues {
```

Comment:
You are removing quite a lot of functionality without even delivering an alternative here.

Reply:
This is just for the `--alice` or is there something else with this? This struct is private.

Reply:
This is used by `--alice`, `--bob`, etc. These `conflicts_with` are also important.